frontend/src/citizen-frontend/calendar/CalendarPage.tsx

```diff
@@ -155,6 +175,7 @@ const CalendarPage = React.memo(function CalendarPage() {
           availableChildren={response.children}
           onReload={loadDefaultRange}
           reservableDays={response.reservableDays}
+          firstReservableDate={calculateFirstReservableDate}
```

Comment:
Is there a reason to pass a function instead of evaluating it here? If yes, I would rename the prop e.g. as `getFirstReservableDate` or maybe `getDefaultStartDate` for logic independent abstraction.